practice-one/src/style.css

```diff
@@ -10,12 +10,14 @@
     --primary-color-200: #F3F0EA;
     --primary-color-300: #66645E; 
     --primary-color-400: #303031;
-    --secondary-color-100: #71A894;
+    --secondary-color-100: #71a89480;
```

Comment:
Color value should be only uppercase or lowercase. Don't use both uppercase and lowercase